stix_shifter_modules/crowdstrike_logscale/stix_transmission/api_client.py

```diff
@@ -0,0 +1,66 @@
+from stix_shifter_utils.stix_transmission.utils.RestApiClientAsync import RestApiClientAsync
+import json
+
+PING_ENDPOINT = 'api/v1/status'
+JOB_ENDPOINT = 'api/v1/repositories'
+
+class APIClient:
+
+    def __init__(self, connection, configuration):
```

Comment:
Set the page size from the config.json in here.

Reply:
this connector doesn't need page size

Reply:
code is updated with page size as suggested